content/manuals/desktop/release-notes.md

```diff
@@ -31,9 +31,9 @@ Take a look at the [Docker Public Roadmap](https://github.com/orgs/docker/projec
 
 ### New
 
-- WSL2 is now faster, more reliable, and has enhanced security
+- Docker Desktop on WSL is now faster, more reliable, and has enhanced security
```

Comment:
Suggestion: 
Existing Docker Desktop installations using the WSL2 engine on Windows are now automatically migrated to a unified single-distribution architecture for enhanced consistency and performance.